pom.xml

```
@@ -42,9 +42,10 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.401.3</jenkins.version>
-    <bom.artifactId>bom-2.401.x</bom.artifactId>
-    <bom.version>2745.vc7b_fe4c876fa_</bom.version>
+    <jenkins.baseline>2.426</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
+    <bom.artifactId>bom-${jenkins.baseline}.x</bom.artifactId>
```

Removing `jenkins.baseline` property as both lines need to be reviewed in case of bump anyway.
```suggestion
    <jenkins.version>2.426.3</jenkins.version>
    <bom.artifactId>bom-2.426.x</bom.artifactId>
```

---

2.426 is fine with current `kubernetes` plugin requirement.